bridges/opentracing/lib/opentelemetry/bridge/opentracing/tracer.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Tracer provides a means of referencing
+      # an OpenTelemetry::Tracer as a OpenTracing::Tracer
+      class Tracer
+        attr_reader :tracer
+
+        def initialize(tracer)
+          @tracer = tracer
+          @tracer_factory = OpenTelemetry::Trace::TracerFactory.new
+        end
+
+        def active_span
+          @tracer.current_span
+        end
+
+        def start_active_span(operation_name,
+                              child_of: nil,
+                              references: nil,
+                              start_time: Time.now,
+                              tags: nil,
+                              ignore_active_scope: false,
+                              finish_on_close: true,
+                              &block)
+          span = @tracer.start_span(operation_name,
+                                    with_parent: child_of,
+                                    attributes: tags,
+                                    links: references,
+                                    start_timestamp: start_time)
+          scope = Scope.new(ScopeManager.new, span, finish_on_close)
```

Comment:
This needs more wrapping & unwrapping. `child_of` could be an OpenTracing Span or a SpanContext, and we need to distinguish the two. We then need to unwrap the OpenTelemetry context before passing it to `@tracer.start_span(... with_parent:`. We need an OpenTracing `Reference` implementation that wraps an OpenTelemetry `Link` or allows "coercing" to a `Link`. The `span` needs to be wrapped in an OpenTracing `Span`.

We should be using the `scope_manager` from this OpenTracing Tracer instance rather than creating a new `ScopeManager` here.